### PR TITLE
As a user, I have Task Groups

### PR DESCRIPTION
**Pull request by dralley**
closes: #6414
https://pulp.plan.io/issues/6414

### CODE REVIEW

**Comment by dralley** [comment]
TODO:

* TaskGroup serializes counts of tasks in each state
* Tests

**Comment by dralley** [comment]
Automated tests for this are just as difficult as for task parent/children, so @bmbouter @dkliban If you want to just merge this with some manual testing, I'll file an issue to eventually write automated tests for both.

File: pulpcore/app/serializers/task.py [hunk 1]
```diff
@@ -117,6 +123,27 @@ class Meta:
                                                 'worker')
 
 
+class TaskGroupSerializer(ModelSerializer):
+    description = serializers.CharField(
+        help_text=_("A description of the task group.")
+    )
+
+    waiting = TaskGroupStatusCountField(state='waiting')
```
Comment (dkliban) [inline pulpcore/app/serializers/task.py:131]: What about some help text for these fields?

**Comment by bmbouter** [comment]
@dralley I'm +1 to merging this with your hand testing done and filing auotmated tests for later.

One idea I had for the automated tests is that we should have some way for them to load additional viewsets and tasks in the RQ registry if there is a PULP_TEST env var set or something. Something that keeps them unloaded on production systems and the tests automatically skip if they aren't loaded. Just an idea I had.

**Pull request merged**